language/resource.language.ar_sa/strings.po

```diff
@@ -2089,13 +2089,18 @@ msgstr "FlixArt حجم صورة الفيديو بالواجهة الرئيسية
 #: /1080i/Includes_Labels.xml
 msgctxt "#31442"
 msgid "Parent"
-msgstr "الام
+msgstr "الاقسام الرئيسية"
 
 #: /1080i/Custom_1116_Dialog_Shortcuts.xml
 msgctxt "#31443"
 msgid "Open path in dialog"
 msgstr """
 
+#: /1080i/SkinSettings.xml
+ msgctxt "#31446"
+ msgid "Bump"
+ msgstr "لا شئ"
```

Comment:
You've got a leading space on these lines which is why it is get highlighted as an error.